src/main/java/nextstep/favorite/application/FavoriteService.java

```diff
@@ -4,43 +4,64 @@
+import nextstep.subway.entity.Station;
+import nextstep.subway.service.LineService;
+import nextstep.subway.service.StationService;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
 
 @Service
 public class FavoriteService {
     private FavoriteRepository favoriteRepository;
+    private StationService stationService;
+    private MemberService memberService;
+    private LineService lineService;
 
-    public FavoriteService(FavoriteRepository favoriteRepository) {
+
+    public FavoriteService(FavoriteRepository favoriteRepository, StationService stationService, MemberService memberService, LineService lineService) {
         this.favoriteRepository = favoriteRepository;
+        this.stationService = stationService;
+        this.memberService = memberService;
+        this.lineService = lineService;
     }
 
-    /**
-     * TODO: LoginMember 를 추가로 받아서 FavoriteRequest 내용과 함께 Favorite 를 생성합니다.
-     *
-     * @param request
-     */
-    public void createFavorite(FavoriteRequest request) {
-        Favorite favorite = new Favorite();
+    /** 즐겨찾기 등록 */
+    public void createFavorite(LoginMember loginMember, FavoriteRequest request) {
+        Member member = memberService.findMemberByEmail(loginMember.getEmail());
+        Station sourceStation = stationService.findStation(request.getSource());
+        Station targetStation = stationService.findStation(request.getTarget());
+        List<Sections> sectionsList = lineService.findSectionsList();
+
+        Favorite favorite = new Favorite(member, sourceStation, targetStation, sectionsList);
         favoriteRepository.save(favorite);
     }
 
-    /**
-     * TODO: StationResponse 를 응답하는 FavoriteResponse 로 변환해야 합니다.
-     *
-     * @return
-     */
-    public List<FavoriteResponse> findFavorites() {
-        List<Favorite> favorites = favoriteRepository.findAll();
-        return null;
+    /** 즐겨찾기 목록 조회 */
+    public List<FavoriteResponse> findFavorites(LoginMember loginMember) {
+        List<Favorite> favorites = favoriteRepository.findAllByMemberEmail(loginMember.getEmail());
+
+        return favorites.stream()
+            .map(it -> new FavoriteResponse(it.getId(), it.getSourceStation(), it.getTargetStation()))
+            .collect(Collectors.toList());
     }
 
-    /**
-     * TODO: 요구사항 설명에 맞게 수정합니다.
-     * @param id
-     */
-    public void deleteFavorite(Long id) {
-        favoriteRepository.deleteById(id);
+    /** 즐겨찾기 삭제 */
+    public void deleteFavorite(LoginMember loginMember, Long id) {
+        Optional<Favorite> deleteFavorite = favoriteRepository.findAllByMemberEmail(loginMember.getEmail()).stream()
+            .filter(it -> it.getId().equals(id))
+            .findAny();
+
+        if(deleteFavorite.isEmpty()) {
+            throw new IllegalArgumentException("등록하지 않은 즐겨찾기는 삭제할 수 없다");
+        }
+
+        favoriteRepository.delete(deleteFavorite.get());
     }
```

Comment:
큰 차이는 없지만 이렇게도 해볼 수 있겠네요 :) 

```suggestion
    public void deleteFavorite(LoginMember loginMember, Long id) {
        Favorite deleteFavorite = favoriteRepository.findAllByMemberEmail(loginMember.getEmail()).stream()
              .filter(it -> it.getId().equals(id))
              .findAny()
              .orElseThrow(() -> new IllegalArgumentException("등록하지 않은 즐겨찾기는 삭제할 수 없다"));

        favoriteRepository.delete(deleteFavorite);
    }
```